### PR TITLE
[util] NVIDIA_TESLA_A100 -> NVIDIA_AMPERE_A100

### PR DESCRIPTION
**Pull request by shayanhoshyari**
## Why are these changes needed?

One minor issue I noticed in this file: https://github.com/ray-project/ray/blob/master/python/ray/util/accelerators/accelerators.py#L6
A100 is not an NVIDIA Tesla architecture processor [it is the new Ampere architecture.](https://www.nvidia.com/en-us/data-center/a100/)

Here is a PR to add the corrected name (I kept. old name for backward compatibility) 

## Related issue number

N/A

## Checks

- [x] I've signed off every commit(by using the -s flag, i.e., `git commit -s`) in this PR.
- [x] I've run `scripts/format.sh` to lint the changes in this PR.
- [x] I've included any doc changes needed for https://docs.ray.io/en/master/.
    - [x] I've added any new APIs to the API Reference. For example, if I added a 
           method in Tune, I've added it in `doc/source/tune/api/` under the 
           corresponding `.rst` file.
- [ ] I've made sure the tests are passing. Note that there might be a few flaky tests, see the recent failures at https://flakey-tests.ray.io/
- Testing Strategy
   - [ ] Unit tests
   - [ ] Release tests
   - [ ] This PR is not tested :(

### CODE REVIEW

**Comment by scottsun94** [comment]
cc: @xieus to triage

**Comment by scottsun94** [comment]
cc: @xieus @richard4912

**Comment by scottsun94** [comment]
cc: @richardliaw 

(@richard4912 sorry for the wrong ping)

**Review by richardliaw — APPROVED** [review]

**Comment by zhe-thoughts** [comment]
Could either @richardliaw or maybe someone from Ray Core team help merge this? @jjyao ?

File: python/ray/util/accelerators/accelerators.py [hunk 1]
```diff
@@ -3,5 +3,10 @@
 NVIDIA_TESLA_T4 = "T4"
 NVIDIA_TESLA_P4 = "P4"
 NVIDIA_TESLA_K80 = "K80"
-NVIDIA_TESLA_A100 = "A100"
 NVIDIA_TESLA_A10G = "A10G"
+
+NVIDIA_AMPERE_A100 = "A100"
```
Comment (jjyao) [inline python/ray/util/accelerators/accelerators.py:8]: Sorry for missing this PR. Could you expose this symbol in `python/ray/util/accelerators/__init__.py`

**Review by jjyao — COMMENTED** [review]
Thanks for the contribution!!

File: python/ray/util/accelerators/accelerators.py (hunk 1 above)
Comment (rkooo567) [reply python/ray/util/accelerators/accelerators.py:8]: +1

**Review by rkooo567 — CHANGES_REQUESTED** [review]

**Comment by rkooo567** [comment]
Please expose the constant to the init file!

**Comment by xieus** [comment]
@shayanhoshyari for awareness.

**Comment by scottsun94** [comment]
Ping for awareness @shayanhoshyari

**Comment by anyscalesam** [comment]
I have context here and can push it through

**Pull request closed without merging**